### DISCUSSION TITLE
Splitting Stereo and L/R EQ?

### DISCUSSION THREAD

**Discussion by Wascha** [wwmm/easyeffects#3285] [Ideas]
I hope I'm not making too much of a fool of myself by requesting something that is already there, but I did play around and look through manuals and documentation and came up with nothing.

As of late I've encountered a few scenarios where I wanted to make a quick EQ change, e.g. when trying out headphones and experimenting with giving a cut or boost here or there. To do that effectively, I would always prefer to use a spectrum graph parametric EQ.
This is of course kind of provided by the "Show Native Window" option, which in itself works just fine. 

My issue lies with the fact that LSP's PE32LR EQ does not allow jointly adjusting, or even copying over, filters on the left and right channel natively. This means they need to be either done swiftly by dragging inside the GUI, which will always lead to at least small mismatches between left and right, or by copying over the values from one filter to the other manually, which is very slow. Both of these options are really not ideal for precise adjustments and comparisons on the fly.

I see two possible ways out of this: 
1) LSP ads a way to jointly adjust filters between channels inside the plugin
2) Splitting the current "Equalizer" effect in EasyEffects in two: "Equalizer L/R" using PE32LR for those who are using the current "Split Channels" option, and "Equalizer Stereo" using PE32S.

I realize of course that this is kind of a niche thing to request, however, to me the purpose of the native EQ plugin window seems kind of defeated for most people who would want to use it in its current state.

Again, apologies if this is already possible to do but my search yielded no results.

**Comment by wwmm** [comment]
>My issue lies with the fact that LSP's PE32LR EQ does not allow jointly adjusting, or even copying over, filters on the left and right channel natively.

As they have a dedicated plugin where the settings are applied to both channels the variant we use probably will never have this feature. The ideal solution would be to wrap the others in EasyEffects. The problem is justifying this additional maintenance burden to fix what is really a corner case. Wrapping LSP plugins require a large amount of work because they have a lot of features.

>Again, apologies if this is already possible to do but my search yielded no results.

It is not possible in a straightforward way. At this moment the less painful approach would be to keep the native window open and using EasyEffects custom widgets to apply the settings while `Split Channels` is disabled. As EasyEffects does the copying from the left channels settings to the right channel you would be able to setting things once.
